Freshly/Behave
# Performance property

**What does this do?** 
It adds a property called **testPerformance*** to Behave. The intention is multi-fold. 

1. Make it simple to turn on and off performance testing for the developer
2. Add performance testing to the entire suite as opposed to forcing the developer to add it in spots.
3. Associate the events being tested with the slow frame renders in order to help the developer more easily debug their code.
4. Adds a new callback to the run() method, **warn**. The warn callback returns a list of events that were being tested when slow frame renderings occurred.
5. Updates ReadMe to include the new functionality.

By default performance testing is off

*note:* I added cleaned up the ReadMe a bit as well as adding info about this feature.

**To Test**

- Run PerformanceBDDTests
- There are three tests: efficient, inefficient with property on,  inefficient with property off. 
- Examine the console post test run and you will see in the **warn** completion handler a list of events returned as a string that are ties to the slow frame rendering on *inefficient with property on* 
- You should see no warnings on the other two. 
- Also, on the test with "property off" it will have slow renders, but will only fail if the warning is triggered. It should not as the property is set to false in the test .
